mscore/editstaff.ui

```diff
@@ -422,16 +512,71 @@
           </property>
          </widget>
         </item>
-        <item row="1" column="1">
-         <widget class="QTextEdit" name="shortName">
-          <property name="minimumSize">
-           <size>
-            <width>0</width>
-            <height>40</height>
-           </size>
+        <item row="3" column="0">
+         <widget class="QLabel" name="labelOffsetX">
+          <property name="toolTip">
+           <string>Negative values move left, positive values move right (-200, 200).</string>
           </property>
-          <property name="tabChangesFocus">
-           <bool>true</bool>
+          <property name="text">
+           <string>Name Offset X:</string>
+          </property>
+         </widget>
+        </item>
+        <item row="2" column="0">
+         <widget class="QLabel" name="rotationLabel">
+          <property name="toolTip">
+           <string>Negative values rotate counterclowise, positive values clockwise (-180...180).</string>
```

Comment:
Typo "counterclowise". Also not sure whether we want to show the limits, but if then with a "°" sign

Reply:
I believe that displaying the limits will help. And you are right about the sign, I'll add units in all of them.